setup/configure_tpm_emulator/swtpm_permissive.te

```diff
@@ -13,6 +13,7 @@ permissive swtpm_t;
 init_daemon_domain(swtpm_t, swtpm_exec_t);
 
 dontaudit swtpm_t self:tcp_socket { accept listen };
+dontaudit swtpm_t swtpm_t:capability { sys_admin };
```

Comment:
I'm not sure about the impact of this change. @Koncpa Could you please check this?

Reply:
For swtpm we have a custom SELinux module that runs swtpm in a permissive domain and we don't audit all AVCs. The reason for that is basically that our swtpm usage is slightly different from the general perception. While starting TPM on a running system is possible, the major use case for swtpm is rather emulating TPM in virtual systems and the existing SELinux policy follows that approach.